src/app/authentication/request-code.ts

```diff
@@ -98,13 +103,18 @@ export const requestPhoneCodeForAuthedUser = async ({
     return new PhoneAlreadyExistsError()
   }
 
-  const testAccounts = getTestAccounts()
-  if (TestAccountsChecker(testAccounts).isPhoneValid(phone)) {
+  // return true if we are in dev or test mode
+  if (!isProd) {
```

Comment:
are we sure this will not affect staging?

Reply:
Good point

@bodymindarts suggested a `--dev` to enable this behavior to be more explicit

Any thought on how to best handle this otherwise?

I don't think we should introduce the pattern of using command line arguments, it's not super typical for a node js app afaik

Otherwise I could add a enable_account_creation_with_defaut_code env variable or something like that